arcgis-ios-sdk-samples/Maps/Set min-max scale/README.md

```diff
@@ -1,22 +1,29 @@
-# Set min/max scale
+# Min max scale
```

Comment:
```suggestion
# Set min/max scale
```
Revert change to title.

I hesitate on this one because Mike mentioned the slash in the title caused the sample not to display on the developer site right now. Should we consider fix this issue as a part of the README updates or later?